release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -80,6 +80,10 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 #### Ballerina Update Tool
 
+#### Debugger
+
+* Introduced the run in terminal feature. This provides the capability to debug Ballerina programs that take user inputs, in the launch mode. This can be achieved by setting the `terminal` attribute in the launch configurations to `integrated`. Once the configuration is set, launching the debugger will cause the Ballerina programs to be executed in a separate integrated VSCode terminal.
```

Comment:
```suggestion
* Introduced the `run` functionality in the terminal feature. This provides the capability to debug Ballerina programs that take user inputs in the launch mode. This can be achieved by setting the `terminal` attribute in the launch configurations to `integrated`. Once the configuration is set, launching the debugger will cause the Ballerina programs to be executed in a separate integrated VS Code terminal.
```

Reply:
Was using the term `run in terminal` to maintain consistency with the protocol phrases: https://microsoft.github.io/debug-adapter-protocol/specification#Reverse_Requests_RunInTerminal

Would it be alright to deviate from this?

Looping in @NipunaRanasinghe for input.

Reply:
Yes we better keep the feature name as it is, as its been commonly used by the debugger protocol as well as the VSCode community. Probably we can do something like the below to emphasise that this is the feature name

```
* Introduced the `run-in-terminal` feature which provides the capability to debug Ballerina programs that take user inputs, in the launch mode. This can be achieved by setting the `terminal` attribute in the launch configurations to `integrated`. Once the configuration is set, launching the debugger will cause the Ballerina programs to be executed in a separate integrated VSCode terminal.
```

Reply:
+1 for this with the changes below:

* Introduced the `run-in-terminal` feature, which provides the capability to debug Ballerina programs 
(that take user inputs) in the launch mode. This can be achieved by setting the `terminal` attribute in 
the launch configurations to `integrated`. Once the configuration is set, launching the debugger will 
cause the Ballerina programs to be executed in a separate integrated VS Code terminal.

Reply:
Modified accordingly. :)